<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiceWindow</class>
 <widget class="QWidget" name="DiceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiceWindow</string>
  </property>
  <widget class="QLineEdit" name="numLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="rollButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="handicapLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QListWidget" name="historyListWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="historyLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Roll History</string>
   </property>
  </widget>
  <widget class="QLabel" name="numLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Number of dices</string>
   </property>
  </widget>
  <widget class="QLabel" name="dLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>135</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Handicap</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="resultlcdNumber">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
